### ISSUE TITLE
Need more info on how to run the api and services to be created

### ISSUE THREAD

**Issue by rachitharlalka** [Azure-Samples/communication-services-contoso-med-app#3]
I am unable to run all features of app like chat bot conversation, video call, doctor-patient call. Need help/more info on the services to be created as pre-requisites for running the application

**Comment by poiriersimon** [comment]
Sames, it appears a lot of the prerequisite aren't documented.

**Comment by ddematheu2** [comment]
Thank you for providing feedback. Did you follow the steps here: https://github.com/Azure-Samples/communication-services-contoso-med-app#set-up. (Set Up section of README contains additional information to setup the different pieces)

**Comment by poiriersimon** [comment]
Yes, but there no information on the QNAbot requirement and source, nor what the JWT token is, Cosmo DB requirement, etc... 
Maybe more since I'm kinda stuck now.

**Comment by poiriersimon** [comment]
@ddematheu2 do you have an update for us?
Thanks

**Comment by ddematheu2** [comment]
Thanks for the additional input. Adding to the loop the team behind the app. @gc-codes  can you help poiriersimon with his request.

**Comment by ddematheu2** [comment]
Additional details were added.

**Comment by poiriersimon** [comment]
Hi @ddematheu2 I don't see any change and all the readme files are from 1 to 4 months. 
Nothing more about what are expected value for the SPOOL or JWT Token in those config files
